Dockerfiles/agent/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # Preparation stage: extract and cleanup #
 ##########################################
 
-FROM debian:buster-slim as extract
+FROM ubuntu:21.04 as extract
```

Comment:
Why Ubuntu (move away from Debian), and not just use Debian Bullseye?
https://hub.docker.com/layers/debian/library/debian/bullseye-slim/images/sha256-fd9bc56e83c76803d355ddbe0a4e18aae08f4ee16d05a6fafb843ee90cc0104c?context=explore

Reply:
`docker pull debian:bullseye-slim`